src/main/java/org/opentripplanner/model/TimetableSnapshot.java

```diff
@@ -183,23 +217,79 @@ public boolean hasRealtimeAddedTripPatterns() {
     return !realtimeAddedTripPattern.isEmpty();
   }
 
+  /**
+   * Return the route created by the updater for the given id.
+   */
+  @Nullable
+  public Route getRealtimeAddedRoute(FeedScopedId id) {
+    return getByNullableKey(id, realtimeAddedRoutes);
+  }
+
+  public Collection<Route> getAllRealTimeAddedRoutes() {
+    return Collections.unmodifiableCollection(realtimeAddedRoutes.values());
+  }
+
+  /**
+   * Return the trip created by the updater for the given id.
+   */
+  @Nullable
+  public Trip getRealTimeAddedTrip(FeedScopedId id) {
+    return getByNullableKey(id, realTimeAddedTrips);
+  }
+
+  public Collection<Trip> getAllRealTimeAddedTrips() {
```

Comment:
This is not following the naming-conventions, it should be `listRealTimeAddedTrips()`. We want to merge this PR, so we do NOT do any minor changes now, but instead do it in a follow up PR. The PR should fix all `get[All]X() : Collection/List`.

Reply:
I added this to https://github.com/opentripplanner/OpenTripPlanner/issues/6048